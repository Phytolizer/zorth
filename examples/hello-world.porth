include "io.porth"

"Hello, World\n" write
