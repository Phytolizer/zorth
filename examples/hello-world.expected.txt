:i returncode 0
:i argc 0
:b stdin 0

:b stdout 13
Hello, World

:b stderr 0

